boule/_triaxialellipsoid.py

```diff
@@ -325,6 +341,27 @@ def meridional_flattening(self):
         """
         return (self.semimajor_axis - self.semiminor_axis) / self.semimajor_axis
 
+    def __str__(self):
+        s = self.name + " - " + self.long_name + "\n"
+        s += "Triaxial ellipsoid:\n"
+        s += f"  Semimajor axis: {self.semimajor_axis} m\n"
+        s += f"  Semimedium axis: {self.semimedium_axis} m\n"
+        s += f"  Semiminor axis: {self.semiminor_axis} m\n"
+        s += f"  Semiminor axis longitude: {self.semimajor_axis_longitude}\n"
```

Comment:
Minor correction here:
```suggestion
        s += f"  Semimajor axis longitude: {self.semimajor_axis_longitude} °\n"
```